addon/components/ember-th/component.js

```diff
@@ -61,6 +61,8 @@ export default BaseTableCell.extend({
 
   rowMeta: readOnly('api.rowMeta'),
 
+  tabindex: 0,
```

Comment:
If this value is only ever `0`, then in `didInsertElement` can you use `setAttribute(` to set it instead of a binding? A binding has setup/memory overhead, a static attribute setup would be better for performance (and not really more confusing or anything).